yt/data_objects/data_containers.py

```diff
@@ -207,6 +207,11 @@ def _set_center(self, center):
                 self.center = self.ds.find_min(center[4:])[1]
         else:
             self.center = self.ds.arr(center, "code_length", dtype="float64")
+
+        if self.center.ndim > 1:
+            mylog.warning("Removing singleton dimensions from 'center'.")
```

Comment:
Not that crucial a message that it warrants a warning, imo
```suggestion
            mylog.debug("Removing singleton dimensions from 'center'.")
```